themes/classic/templates/catalog/_partials/product-prices.tpl

```diff
@@ -97,6 +97,7 @@
       {/if}
       {hook h='displayProductPriceBlock' product=$product type="price"}
       {hook h='displayProductPriceBlock' product=$product type="after_price"}
+      {if $product.is_virtual	== 0}
       {if $product.additional_delivery_times == 1}
```

Comment:
Must be indented otherwise it's confusing

Reply:
changes indented.